tests/geomultiscale-mapping/AxialGeoMultiscale.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration dimensions="3" experimental="on">
+  <data:vector name="DataOne" />
+  <data:vector name="DataTwo" />
+
+  <mesh name="MeshOne">
+    <use-data name="DataOne" />
+    <use-data name="DataTwo" />
+  </mesh>
+
+  <mesh name="MeshTwo">
+    <use-data name="DataOne" />
+    <use-data name="DataTwo" />
+  </mesh>
+
+  <participant name="SolverOne">
+    <provide-mesh name="MeshOne" />
+    <write-data name="DataOne" mesh="MeshOne" />
+    <read-data name="DataTwo" mesh="MeshOne" />
+  </participant>
+
+  <participant name="SolverTwo">
+    <receive-mesh name="MeshOne" from="SolverOne" />
+    <provide-mesh name="MeshTwo" />
+    <mapping:axial-geometric-multiscale
+      direction="write"
+      multiscale-type="spread"
+      multiscale-radius="1.0"
+      multiscale-axis="Z"
+      from="MeshTwo"
+      to="MeshOne"
+      constraint="consistent" />
+    <mapping:axial-geometric-multiscale
+      direction="read"
+      multiscale-type="spread"
+      multiscale-radius="1.0"
+      multiscale-axis="Z"
+      from="MeshOne"
+      to="MeshTwo"
```

Comment:
Looking at the bigger picture, I think we should remove the `spread`/`collect` from the config, as it is confusing, and preCICE has all information it needs to classify the mapping (see #1725).

The two terms may also be a bit confusing, since they signify direction of information between meshes, while we mainly want to signify direction of dimensions conversion (from lower to higher, or from higher to lower).

Looking at the specific example, we need to take into account that both mappings happen in the same participant:

- Write mapping from MeshTwo (3D) to MeshOne (1D), which means 3D->1D, i.e., collect
- Read mapping from MeshOne (1D) to MeshTwo (3D), which means 3D<-1D, i.e., spread

@ezonta I think that the comment is right: the write mapping should be a "collect". But the terms are confusing when combined with "read"/"write".

Reply:
If I set the write mapping to `collect`, then the test is failing (correctly):

```
error: in "Integration/GeometricMultiscale/AxialGeoMultiscale": check valueDataB == expected has failed
Failure occurred in a following context:
    Test context represents "SolverOne" and runs on rank 0 out of 1.
```

Reply:
To remove the `spread`/`collect` from the config, we first need the 1D mode: https://github.com/precice/precice/issues/1669

Reply:
Update on my previous description:

- We currently don't have support for 1D meshes, so both meshes are 3D.
- The axial mappings internally assume the dimensions based on the type of the mapping (spread/collect).
- Both mappings happen in `SolverTwo`, who provides `MeshTwo`.
- First mapping:
   - Mapping from `MeshTwo` to `MeshOne`, defined as spread, so the `from` (input) mesh `MeshTwo` is assumed to be 1D, and the `to` (output) mesh `MeshOne` is assumed to be 3D.
   - Since it is from `MeshTwo` (own) to `MeshOne` (other), it is a write mapping.
- Second mapping:
   - Mapping from `MeshOne` to `MeshTwo`, defined as spread, so the `from` (input) mesh `MeshOne` is assumed to be 1D, and the `to` (output) mesh `MeshTwo` is assumed to be 3D.
   - Since it is from `MeshOne` (other) to `MeshTwo` (own), it is a read mapping.

Clearly a problem of not having explicit distinction between 1D and 3D meshes at the moment.